### PR TITLE
Update suffix list for determining valid TLDs

### PR DESCRIPTION
**Pull request by thunderd0m3**
### Description
This code is validating TLDs against a public suffix list, which can be obtained from here: https://publicsuffix.org/list/

This PR is updating the existing list to the most current one on the public suffix site.

### CODE REVIEW

File: data/public_suffix_list.dat [hunk 1]
```diff
@@ -2,6 +2,11 @@
 // License, v. 2.0. If a copy of the MPL was not distributed with this
 // file, You can obtain one at https://mozilla.org/MPL/2.0/.
 
+// Please pull this list from, and only from https://publicsuffix.org/list/public_suffix_list.dat,
```
Comment (bemurphy) [inline data/public_suffix_list.dat:5]: nice addition!

File: data/public_suffix_list.dat [hunk 2]
```diff
@@ -8773,10 +8883,16 @@ living
 // lixil : 2015-03-19 LIXIL Group Corporation
 lixil
 
+// llc : 2017-12-14 Afilias Limited
+llc
```
Comment (bemurphy) [inline data/public_suffix_list.dat:8887]: and here's the llc we were missing! Nice

**Review by bemurphy — APPROVED** [review]
👍 LGTM, I see it has `llc` in there, which triggered this change, which is great.  I'm probably being overly cautious, but throwing out there anyway just in case. Do you think there's any value getting our list of custom domains, and then running against this gem with this update (outside production) to see how it stacks up? Like I said, probably overkill, but might be a good fail-safe check to consider.

**Comment by thunderd0m3** [comment]
> 👍 LGTM, I see it has `llc` in there, which triggered this change, which is great. I'm probably being overly cautious, but throwing out there anyway just in case. Do you think there's any value getting our list of custom domains, and then running against this gem with this update (outside production) to see how it stacks up? Like I said, probably overkill, but might be a good fail-safe check to consider.

Yes I think that's probably a smart idea 👍

**Review by cjbuchmann — APPROVED** [review]
very nice, thank you! 🚀

**Pull request merged**